README.md

```diff
@@ -191,7 +219,16 @@ Attributes
  * `node["kafka"]["offset_monitor"]["db_name"]` : The base file name for the offset monitoring database file written into the kafka user's home directory (default = "offset_monitor")
  * `node["kafka"]["service"]["stdout"]` : The file to keep std output of kafka init service (default = "/dev/null")
  * `node["kafka"]["service"]["stderr"]` : The file to keep std error of kafka init service (default = "/dev/null")
-
+ * `node["kafka"]["mirror_maker"]["whitelist"]` : A whitelist of topics that will be mirrored. (default = ".*")
+ * `node["kafka"]["mirror_maker"]["blacklist"]` : A blacklist of topics that will not be mirrored. (default = nil)
+ * `node["kafka"]["mirror_maker"]["streams"]` : Number of consumer threads to run. (default = 2)
+ * `node["kafka"]["mirror_maker"]["mirror_target.properties"][*]` : A key/value that will be set in mirror's target properties file. Used to customize the mirror configuration. See [Mirror docs](https://kafka.apache.org/documentation.html#basic_ops_mirror_maker)
+ * `node["kafka"]["mirror_maker"]["mirror_sources"]` : a list of source property files to create. (default = ["mirror_source1.properties"])
```

Comment:
Would it make more sense to make this a hash where the keys could still be the property files or names of source clusters and value could be a hash of values (i.e. properties)?

    kafka => {
      mirror_maker => {
        mirror_sources => {
          src_cluster_a => {
            property1 => value1
            ...
          },
          src_cluster_2 => {
            property1 => value1,
            ...
          }
        }
      }
    }